utils/utils.js

```diff
@@ -420,6 +420,30 @@ function getUserProvidedConfigUrl(configUrl) {
   return url;
 }
 
+function recurse(cur, prop, result) {
```

Comment:
Add a comment explaining what this method does

Reply:
Fixed